### DISCUSSION TITLE
路由器多个LAN网卡如何绑定？

### DISCUSSION THREAD

**Discussion by 0xlane** [vernesong/OpenClash#2589] [Q&A]
openclash默认绑定wan网卡，由于是做透明代理openclash不能打开socks5/http代理认证，这就导致openclash的所有开放的端口暴露在外网，所以我想最直接的办法是修改openclash的绑定网卡，但是我有两个内网LAN网卡，如下图，选择其中1个试了一下发现上不了网了：
![image](https://user-images.githubusercontent.com/22788893/181908719-ca39f248-f5a8-4d15-8730-cc06534bdb77.png)
现在只能通过防火墙去限制这些端口，比较麻烦。
![image](https://user-images.githubusercontent.com/22788893/181908920-2c4e18ed-d927-422b-ac08-37a906f275b7.png)

想问一下是我配置的方法不正确还是bug？

**Comment by vernesong** [comment]
![image](https://user-images.githubusercontent.com/42875168/182784532-00ffbe62-951e-4f6b-895b-3c6f48eb6950.png)

**Reply by 0xlane** [reply]
多lan，不是多个wan

**Comment by vernesong** [comment]
开仅允许内网或者上面这样设置不就行了吗
